src/app/pages/fund/fiat-providers-list.tsx

```diff
@@ -30,14 +29,7 @@ export function FiatProvidersList(props: FiatProvidersProps) {
   const analytics = useAnalytics();
   const location = useLocation();
 
-  const routeToQr = useMemo(() => {
-    switch (symbol) {
-      case 'BTC':
-        return RouteUrls.ReceiveBtc;
-      case 'STX':
-        return RouteUrls.ReceiveStx;
-    }
-  }, [symbol]);
+  const routeToQr = symbol === 'BTC' ? RouteUrls.ReceiveBtc : RouteUrls.ReceiveStx;
```

Comment:
👏

Reply:
Not big on this change. 

Consider we add `ETH`. `ETH !== 'BTC'` so it's the STX route here. This approach is more prone to error than previous.

Reply:
I can add the route to the one switch I refactor too, but I'm not a fan of keeping several switch statements to return very little data.

Reply:
Suggest a map, instead then

```ts

const qrCodeRouteMap = {
  BTC: RouteUrls.ReceiveBtc,
  STX: RouteUrls.ReceiveStx
};
```

?

Reply:
Ok, good idea, will do.

Reply:
It screams `enum` to me? 🍿